jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/sso/bearer/ConfirmUserCodeLoginDialog.kt

```diff
@@ -68,6 +75,28 @@ class ConfirmUserCodeLoginDialog(
         super.doCancelAction()
         AwsTelemetry.loginWithBrowser(project = null, Result.Cancelled, credentialType)
     }
+
+    override fun createActions(): Array<Action> {
+        val defaultButtons = arrayOf(cancelAction, okAction)
+        return if (ApplicationInfo.getInstance().build.productCode != "GW") {
+            defaultButtons
+        } else {
+            val activeConnection = CodeCatalystCredentialManager.getInstance(null).connection()
+            if (activeConnection != null) {
+                arrayOf(SignInWithDifferentAccountButton(), cancelAction, okAction)
+            } else {
+                defaultButtons
+            }
+        }
+    }
```

Comment:
we don't need to pass in the indicator and override the action behavior, just check if the dialog exit code in `tokenPending`